stable/kube2iam/templates/daemonset.yaml

```diff
@@ -59,7 +60,19 @@ spec:
               value: {{ .Values.aws.region }}
           {{- end }}
           ports:
-            - containerPort: 8181
+            - containerPort: {{ .Values.host.port }}
+          {{- if .Values.livenessProbeEnabled }}
```

Comment:
Why would you make this optional?

Reply:
Hey @unguiculus - was just leaving another option for users to tweak as needed but on reflection, I can't think of any _sensible_ reason why anyone would want the liveness probe disabled so have removed.